machine/delta/delta_wb2448/machine.make

```diff
@@ -0,0 +1,38 @@
+# Delta Electronics, Inc.
+# CPU Module: Intel Atom BayTrail
+
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = bcm
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# Delta Electronics, Inc. IANA number
+VENDOR_ID = 2254 
+
+# This platform requires the PXE_EFI64 installer
+PXE_EFI64_ENABLE = yes
+
+# Default to msdos disk label for this platform
+PARTITION_TYPE = gpt
+
+# Console parameters
+CONSOLE_DEV = 0
+CONSOLE_FLAG = 1
```

Comment:
`CONSOLE_FLAG` is no longer a parameter -- it has no meaning.  Please remove this line.